leo_description/hooks/leo_description.sh.in

```diff
@@ -0,0 +1 @@
+ament_prepend_unique_value IGN_GAZEBO_RESOURCE_PATH "$AMENT_CURRENT_PREFIX/share/@PROJECT_NAME@/models"
```

Comment:
Since the dsv file takes precedence I suggest not adding the shell script hook, especially since it does not seem to work correctly (it added `/opt/ros/humble/share/leo_description/models` path when I tried it)

```suggestion
```